Week1/homework/todo.sql

```diff
@@ -0,0 +1,74 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+-- MY Database 
+
+-- -----------------------------------------------------
+-- Schema mydb
+--
+-- MY Database 
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 COLLATE utf8_bin ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Users` (
+  `idUsers` INT NOT NULL,
+  `First name` VARCHAR(45) NOT NULL,
```

Comment:
It is better not to use spaces in the column names, and to use all lowercase letters. Instead of `First name`, it is better to call it first_name. To see why, try to write "SELECT First name FROM Users" on your database.